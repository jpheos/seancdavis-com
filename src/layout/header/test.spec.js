import React from "react"
import renderer from "react-test-renderer"

import Header from "./"

// Note: The header brings its own content from the content source, so we just
// need to make sure it can render itself directly.
//
// And therefore we don't need rendering specs for children of this component.

describe("Header", () => {
  it("renders correctly with no props", () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
