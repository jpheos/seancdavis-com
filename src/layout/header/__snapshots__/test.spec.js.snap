// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly with no props 1`] = `
<header
  className="py-4 relative z-10"
>
  <div
    className="container flex justify-between items-center"
  >
    <a
      href="/"
    >
      <svg
        className="h-12"
        viewBox="0 0 57.9999 62"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M36.5683,21.1826l3.6774,9.8061A7.41,7.41,0,0,1,33.3078,41H24a20.0709,20.0709,0,0,1-2.5684-.1826L29,61h4.5207c10.8446,0,20.081-8.4328,20.4663-19.2706A19.99,19.99,0,0,0,36.5683,21.1826Z"
          fill="#2ab6cc"
        />
        <path
          d="M24.6921,21H34a20.0709,20.0709,0,0,1,2.5684.1826L29,1H24.4793C13.6347,1,4.3984,9.4328,4.013,20.2706A19.99,19.99,0,0,0,21.4315,40.8174l-3.6773-9.8061A7.41,7.41,0,0,1,24.6921,21Z"
          fill="#83c6ae"
        />
        <path
          d="M33.3078,41a7.41,7.41,0,0,0,6.9379-10.0113l-3.6774-9.8061A20.0709,20.0709,0,0,0,34,21H24.6921a7.41,7.41,0,0,0-6.9379,10.0113l3.6773,9.8061A20.0709,20.0709,0,0,0,24,41Z"
          fill="#2a7c7c"
        />
        <circle
          cx="11"
          cy="51"
          fill="#2ab6cc"
          r="10"
        />
        <circle
          cx="46.9999"
          cy="11"
          fill="#83c6ae"
          r="10"
        />
      </svg>
    </a>
    <ul
      className="items-center space-x-4 hidden lg:flex"
    >
      <li>
        <a
          className="desktopLink"
          href=""
          rel="noopener"
          target="_blank"
        >
          
        </a>
      </li>
    </ul>
    <button
      className="lg:hidden w-8 text-right"
      onClick={[Function]}
    >
      <span
        className="text-gray-blue h-8 svg-h-full icon"
      >
        <svg
          height={256}
          viewBox="0 0 256 256"
          width={256}
        >
          <path
            d="M227.368421,59.2631579 L28.6315789,59.2631579 C18.341642,59.2631579 10,50.9215159 10,40.6315789 C10,30.341642 18.341642,22 28.6315789,22 L227.368421,22 C237.658358,22 246,30.341642 246,40.6315789 C246,50.9215159 237.658358,59.2631579 227.368421,59.2631579 Z"
          />
          <path
            d="M227.368421,146.210526 L28.6315789,146.210526 C18.3416423,146.210526 10.0000008,137.868884 10.0000008,127.578947 C10.0000008,117.289011 18.3416423,108.947369 28.6315789,108.947368 L227.368421,108.947368 C237.658358,108.947368 246,117.28901 246,127.578947 C246,137.868884 237.658358,146.210526 227.368421,146.210526 L227.368421,146.210526 Z"
          />
          <path
            d="M227.368421,233.157895 L28.6315789,233.157895 C18.341642,233.157895 10,224.816253 10,214.526316 C10,204.236379 18.341642,195.894737 28.6315789,195.894737 L227.368421,195.894737 C237.658358,195.894737 246,204.236379 246,214.526316 C246,224.816253 237.658358,233.157895 227.368421,233.157895 Z"
          />
        </svg>
      </span>
    </button>
    <div
      className="mobileMenu"
    >
      <div
        className="container mobileHeader"
      >
        <a
          href="/"
        >
          <svg
            className="h-12"
            viewBox="0 0 57.9999 62"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M36.6,21.2l3.7,9.8c1.8,4.8-1.8,10-6.9,10H24c-0.9,0-1.7-0.1-2.6-0.2L29,61h4.5c10.8,0,20.1-8.4,20.5-19.3 C54.4,31.2,46.6,22.5,36.6,21.2z"
              fill="#FFFFFF"
              style={
                Object {
                  "fillOpacity": "0.5",
                }
              }
            />
            <path
              d="M24.7,21H34c0.9,0,1.7,0.1,2.6,0.2L29,1h-4.5C13.6,1,4.4,9.4,4,20.3c-0.4,10.5,7.4,19.3,17.4,20.5L17.8,31 C15.9,26.2,19.5,21,24.7,21z"
              fill="#FFFFFF"
              style={
                Object {
                  "fillOpacity": "0.3",
                }
              }
            />
            <path
              d="M33.3,41c5.2,0,8.8-5.2,6.9-10l-3.7-9.8C35.7,21.1,34.9,21,34,21h-9.3c-5.2,0-8.8,5.2-6.9,10l3.7,9.8 c0.8,0.1,1.7,0.2,2.6,0.2H33.3z"
              fill="#FFFFFF"
              style={
                Object {
                  "fillOpacity": "0.75",
                }
              }
            />
            <circle
              cx="11"
              cy="51"
              fill="#FFFFFF"
              r="10"
              style={
                Object {
                  "fillOpacity": "0.5",
                }
              }
            />
            <circle
              cx="47"
              cy="11"
              fill="#FFFFFF"
              r="10"
              style={
                Object {
                  "fillOpacity": "0.3",
                }
              }
            />
          </svg>
        </a>
        <button
          className="w-8 text-right opacity-50"
          onClick={[Function]}
        >
          <span
            className="text-white h-8 svg-h-full icon"
          >
            <svg
              height={256}
              viewBox="0 0 256 256"
              width={256}
            >
              <path
                d="M226.333333,243.190476 C221.864047,243.186542 217.579247,241.407945 214.420952,238.245714 L17.7542857,41.5790476 C11.5652675,34.9371141 11.7478862,24.5868291 18.1673577,18.1673577 C24.5868291,11.7478862 34.9371141,11.5652675 41.5790476,17.7542857 L238.245714,214.420952 C243.060571,219.241799 244.499898,226.487117 241.893307,232.782281 C239.286716,239.077445 233.146803,243.184483 226.333333,243.190476 Z"
              />
              <path
                d="M29.6666667,243.190476 C22.8531971,243.184483 16.7132839,239.077445 14.1066927,232.782281 C11.5001015,226.487117 12.9394294,219.241799 17.7542857,214.420952 L214.420952,17.7542857 C221.062886,11.5652675 231.413171,11.7478862 237.832642,18.1673577 C244.252114,24.5868291 244.434732,34.9371141 238.245714,41.5790476 L41.5790476,238.245714 C38.4207527,241.407945 34.1359526,243.186542 29.6666667,243.190476 Z"
              />
            </svg>
          </span>
        </button>
      </div>
      <ul
        className="container mobileLinkList"
      >
        <li>
          <a
            className="mobileLink"
            href=""
            rel="noopener"
            target="_blank"
          >
            
          </a>
        </li>
      </ul>
    </div>
  </div>
</header>
`;
